README.md

```diff
@@ -249,9 +249,12 @@ An accelerometer doesn't seem to be available at all.
 
 ## Thunderbolt
 
-![MacBookPro13,1 unknown](https://img.shields.io/badge/MacBookPro13%2C1-unknown-lightgrey.svg) ![MacBookPro13,2 unknown](https://img.shields.io/badge/MacBookPro13%2C2-unknown-lightgrey.svg) ![MacBookPro13,3 unknown](https://img.shields.io/badge/MacBookPro13%2C3-unknown-lightgrey.svg) ![MacBookPro14,1 unknown](https://img.shields.io/badge/MacBookPro14%2C1-unknown-lightgrey.svg) ![MacBookPro14,2 unknown](https://img.shields.io/badge/MacBookPro14%2C2-unknown-lightgrey.svg) ![MacBookPro14,3 unknown](https://img.shields.io/badge/MacBookPro14%2C3-unknown-lightgrey.svg)
+![MacBookPro13,1 unknown](https://img.shields.io/badge/MacBookPro13%2C1-unknown-lightgrey.svg) ![MacBookPro13,2 unknown](https://img.shields.io/badge/MacBookPro13%2C2-unknown-lightgrey.svg) ![MacBookPro13,3 working](https://img.shields.io/badge/MacBookPro13%2C3-working-green.svg) ![MacBookPro14,1 unknown](https://img.shields.io/badge/MacBookPro14%2C1-unknown-lightgrey.svg) ![MacBookPro14,2 unknown](https://img.shields.io/badge/MacBookPro14%2C2-unknown-lightgrey.svg) ![MacBookPro14,3 unknown](https://img.shields.io/badge/MacBookPro14%2C3-unknown-lightgrey.svg)
 
-Untested, but probably working with the following patch:
+Tested on the Macbook13,3 with a HP Thunderbolt 3 Dock. Displayport
+Port with a 4k Display with 60 Hz works.
+
+Untested on other models, but probably working with the following patch:
 https://gist.github.com/l1k/3ffbd63ac337fa85530f6db3e9ee77e9
```

Comment:
These PCI id's (and associated functionality) were added in 4.13, so the patch is definitely not necessary as of 4.13.